Saml (#107)
pluginsGLPI/doc: source/saml/tabs/rules.rst

Rules for assigning authorisations
----------------------------------

It will be necessary to establish rules for assigning authorisations to your users (to give them a profile, for example).

> To do this, go to **Administration** > **Rules** > **Rules for assigning authorisations to a user**.

Reviewer: **Rules for assigning authorisations to a user** is not implemented by the plugin. It uses its own rule set: **Rules** > ** Glpisaml - JIT import rules **

These rules can also be accessed via an additional header button in the plugins main configuration page.

> There are no ‘mandatory’ rules, it’s up to you to create the rule(s) that you feel best suit your needs according to your available criteria.

Reviewer: A hard limitation in the current plugin is that the rules can only be bound to the 'email' condition. We are planning to allow binding it to additional SamlClaims, currently it only allows the value communicated via the nameId property or emailaddress claim.

Rules for assigning authorisations
----------------------------------

It will be necessary to establish rules for assigning authorisations to your users (to give them a profile, for example).

To do this, go to **Administration** > **Rules** > **Rules for assigning authorisations to a user**.

There are no ‘mandatory’ rules, it’s up to you to create the rule(s) that you feel best suit your needs according to your available criteria.

> For example, a very simple rule
> 
> A very simple rule -> I want my users with SSO authentication to obtain the Self-Service profile.
> 
> So I set my criteria and my action :

Reviewer 2: Inserting the writer into documentation seems odd.

```suggestion
For example, you want your users with SSO authentication to obtain the Self-Service profile.

You would set your criteria and action as shown here:

```
